integrationTests/multiShard/endOfEpoch/ExecuteBlocksWithTransactionsAndCheckRewards/ExecuteBlocksWithTransactionsAndCheckRewards_test.go

```diff
@@ -0,0 +1,76 @@
+package ExecuteBlocksWithTransactionsAndCheckRewards
+
+import (
+	"context"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/multiShard/endOfEpoch"
+	"testing"
+	"time"
+)
+
+func TestExecuteBlocksWithTransactionsAndCheckRewards(t *testing.T) {
```

Comment:
can you rename this test ? maybe add a `AtEndOfEpoch` suffix as there already is a test with the same name in a different package.
Also. I don't see any `require` or `assert` used in this test. I know it's not related to this PR, but can you check ?

Reply:
not done yet